metagpt/provider/google_gemini_api.py

```diff
@@ -141,7 +143,11 @@ async def _achat_completion_stream(self, messages: list[dict], timeout: int = US
         )
         collected_content = []
         async for chunk in resp:
-            content = chunk.text
+            try:
+                content = chunk.text
+            except Exception as e:
+                logger.warning(f"messages: {messages}\nerrors: {e}\n{BlockedPromptException(str(chunk))}")
+                raise BlockedPromptException(str(chunk))
```

Comment:
Wouldn't it naturally throw this exception? I wonder what it would throw out naturally?

Reply:
"The exception message has nothing to do with `BlockedPromptException`, it needs to be wrapped:
```text
File "/Users/samsaha2/miniconda3/envs/metagpt/lib/python3.9/site-packages/google/generativeai/types/generation_types.py", line 328, in text
raise ValueError(
ValueError: The response.text quick accessor only works for simple (single-Part) text responses. This response is not simple text.Use the result.parts accessor or the full result.candidates[index].content.parts lookup instead.
```